tests/unit/ops/transformer/inference/test_layer_norm.py

```diff
@@ -86,7 +86,7 @@ def test_layer_norm_residual(batch, seq_len, channels, dtype):
     assert allclose(new_output, ref_output)
 
 
-def residual_store_ref_implementation(vals, bias, res, gamma, beta, espilon, channels, dtype):
+def residual_store_ref_implementation(vals, bias, res, gamma, beta, epsilon, channels, dtype):
```

Comment:
`epsilon` isn't used in this function, could you instead remove it from the args and the function call below? Thanks!

Reply:
Modif done ,please check again